code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -211,51 +206,32 @@
 		return
 	usr.update_inv_wear_suit()
 
-//pyjamas
```

Comment:
removed because duplicate defines, these two are unused and broken, the other pajamas are correctly in clothing/under/misc